src/test/java/uk/gov/hmcts/ccd/endpoint/std/EventsEndpointIT.java

```diff
@@ -58,7 +58,7 @@ public void setUp() {
         mockMvc = MockMvcBuilders.webAppContextSetup(wac).build();
         template = new JdbcTemplate(db);
     }
-
+    
     @Test
```

Comment:
Can you check that IDE does not introduce space or it may fail static code analysis

Reply:
applied formatting